build.gradle

```
@@ -107,6 +107,9 @@ tasks.withType(Test).configureEach {
         maxRetries = isCI ? 1 : 0
         maxFailures = 20
     }
+    predictiveSelection {
+        enabled = providers.gradleProperty("isPTSEnabled").map({ it.toBoolean() }).orElse(false)
```

You can drop the parenthesis for `map`.

```suggestion
        enabled = providers.gradleProperty("isPTSEnabled").map { it.toBoolean() } .orElse(false)
```